Neos.Neos/Classes/Service/UserService.php

```diff
@@ -61,33 +59,6 @@ public function getBackendUser()
         return $this->userDomainService->getCurrentUser();
     }
 
-    /**
-     * Returns the name of the currently logged in user's personal workspace
-     * (even if that might not exist at that time).
-     * If no user is logged in this method returns null.
-     *
-     * @api
-     */
-    public function getPersonalWorkspaceName(): ?string
```

Comment:
despite that being earlier available in `context.userInformation.personalWorkspaceName` in security i think so too that this can go? Especially since security will work differently in Neos 9 right?

Reply:
right, I agree